bitnami/harbor/templates/_helpers.tpl

```diff
@@ -0,0 +1,638 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "harbor.name" -}}
+{{- default "harbor" .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "harbor.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "harbor.fullname" -}}
+{{- $name := default "harbor" .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/* Helm required labels */}}
+{{- define "harbor.labels" -}}
+app.kubernetes.io/name: {{ template "harbor.name" . }}
+helm.sh/chart: {{ template "harbor.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end -}}
+
+{{/* matchLabels */}}
+{{- define "harbor.matchLabels" -}}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/name: {{ template "harbor.name" . }}
+{{- end -}}
```

Comment:
These "matchLabels" are not right, you're telling the deployments, svc, etc. to look for pods with labels (assuming your release is named `my-release`):
```yaml
app.kubernetes.io/managed-by: Tiller
app.kubernetes.io/name: harbor
```
So, if I install again the chart (using release name `my-second-release):
```yaml
app.kubernetes.io/managed-by: Tiller
app.kubernetes.io/name: harbor
```
You have the same labels!!!!

---

You should use the labels below to guarantee they're unique:

```yaml
app.kubernetes.io/name: {{ template "harbor.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
```

Reply:
ohh good catch! Thanks